screens/EditBaniOrder.js

```diff
@@ -64,46 +126,27 @@ class EditBaniOrder extends React.Component {
               name="refresh"
               color={GLOBAL.COLOR.TOOLBAR_TINT}
               size={30}
-              onPress={() => this.props.setBaniOrder(defaultBaniOrderArray)}
+              onPress={() => setBaniOrder(defaultBaniOrderArray)}
             />
           }
         />
-        <View
-          style={[
-            styles.container,
-            this.props.nightMode && { backgroundColor: "#464646" },
-          ]}>
+        <View style={[styles.container, nightMode && { backgroundColor: "#464646" }]}>
           <SortableList
             style={styles.list}
             contentContainerStyle={styles.contentContainer}
-            data={this.props.mergedBaniData.baniOrder}
+            data={mergedBaniData.baniOrder}
             onChangeOrder={(nextOrder) => {
               this.newOrder = nextOrder;
             }}
-            onReleaseRow={(key) =>
-              this.newOrder !== undefined
-                ? this.props.setBaniOrder(this.newOrder)
-                : null
-            }
+            onReleaseRow={() => (this.newOrder !== undefined ? setBaniOrder(this.newOrder) : null)}
+            // eslint-disable-next-line no-underscore-dangle
```

Comment:
Veerji please try to remove this eslint disabled rule 

https://eslint.org/docs/latest/rules/no-underscore-dangle

Reply:
fixed